yb-voyager/cmd/analyzeSchema.go

```diff
@@ -2120,8 +2127,12 @@ func packAndSendAnalyzeSchemaPayload(status string) {
 	var callhomeIssues []utils.Issue
 	for _, issue := range schemaAnalysisReport.Issues {
 		issue.SqlStatement = ""  // Obfuscate sensitive information before sending to callhome cluster
-		issue.ObjectName = "XXX" // Redacting object name before sending
-		for _, sensitiveReason := range reasonsIncludingSensitiveInformation {
+		for _, r := range reasonsToSendObjectNameToCallhome {
```

Comment:
I'm just looking at the logic again, I think it seems wrong. 
```
for _, r := range reasonsToSendObjectNameToCallhome
    if !strings.Contains(issue.Reason, r)
        issue.ObjectName = "XXX"
```

reasonsToSendObjectNameToCallhome is a list. So since issue.Reason can not contain every reason in list, you will always override it to XXX?

Reply:
Oh yes, right. Then ContainsBy looks good and is more readable. Made the change. Thanks